ts/Series/Timeline/TimelineSeries.ts

```diff
@@ -239,12 +239,16 @@ class TimelineSeries extends LineSeries {
 
     public getVisibilityMap(): Array<(boolean|TimelinePoint|TimelinePointOptions)> {
         const series = this,
+            nullInteraction = series.options.nullInteraction,
```

Comment:
Could you add a demo for the API and a test for the timeline series? When using just `null` it fails silently - it works when used like `{y: null}` but that is not very intuicial for a timeline series IMHO.
![image](https://github.com/user-attachments/assets/b062dc10-81d0-4b7d-abd9-3309b5395c19)

Reply:
This is still the case - there's no demo for the timeline series using null points linked in the API docs. It either looks wrong or doesn't seem to be working correctly (tooltip not showing, unable to hover) depending on using `null` or `{y:null}` as a point.